Pyrado/pyrado/algorithms/utils.py

```diff
@@ -269,3 +269,13 @@ def reset_rollouts(self) -> None:
     def __getattr__(self, name: str) -> Any:
         # Forward to the wrapped sampler
         return getattr(self.wrapped_sampler, name)
+
+    def __getstate__(self):
+        """ Required so that pickle does not end in an endless recursion
+        """
+        return vars(self)
+
+    def __setstate__(self, state):
+        """ Required so that pickle does not end in an endless recursion
+        """
```

Comment:
Please move the three quotes to the end of line 274, add a period after the text and remove the surrounding spaces (Black formatting).